### ISSUE TITLE
Returning a config object?

### ISSUE THREAD

**Issue by tannerdolby** [tannerdolby/eleventy-plugin-social-img#1]
Currently the `{% socialImg %}` only returns a URL string containing the image resource generated by `captureWebsite`.

Like: `https://tannerdolby.com/images/some-image.png`

There is a config object that contains all the metadata for the screenshot generated. Would it be worthwhile to have that object logged to the console when images are created for debugging purposes?

```js
{ input: '<h1>Some Post Title</h1>',
  inputType: 'html',
  inputDir: './src',
  width: 600,
  height: 315,
  type: 'png',
  scaleFactor: 2,
  fullPage: false,
  defaultBackground: true,
  delay: 0,
  disableAnimations: false,
  isJavaScriptEnabled: true,
  headers: {},
  debug: false,
  darkMode: false,
  inset: 0,
  launchOptions: {},
  overwrite: true,
  preloadFunction: undefined,
  debugOutput: false,
  styles: [ 'h1{color:#f06;}body{background:green}' ] }
```

My only concern is that since the images are generated on every eleventy build, it could pollute the console with such a large object consuming many lines.

**Comment by tannerdolby** [comment]
I think a good way to provide a "sort of" fix for this, would be to create another shortcode argument that is called `debugOuput` or something of that nature (since `debug` is already used in capture-website) which will log the config object to the console, it will be `false` by default but can be added as `debugOutput=true` to have the config object logged to console when generating images.

**Comment by tannerdolby** [comment]
Fixed with commit [f507aa8](https://github.com/tannerdolby/eleventy-plugin-social-img/commit/f507aa829968e6da47fed03d17de6013a3e011b2)
